app/Http/Controllers/ReviewsController.php

```diff
@@ -51,13 +51,14 @@ public function reviews(ReviewsRequest $request)
         $review['northstar_id'] = $post->northstar_id;
         $review['old_status'] = $post->status;
 
+        // Append admin's ID to the request for the "reviews" service.
+        $review['admin_northstar_id'] = auth()->user()->northstar_id;
         $reviewedPost = $this->post->reviews($review);
         $reviewedPostCode = $this->code($reviewedPost);
-
         $meta = [];
 
         if (isset($reviewedPost)) {
-            return $this->item($reviewedPost, $reviewedPostCode, $meta, $this->postTransformer);
+            return $this->item($reviewedPost, $reviewedPostCode);
         } else {
             return 404;
```

Comment:
Let's update this to throw a `ModelNotFoundException` since I don't think this actually sets a status code.